### PR TITLE
feat(activityCategorization): include 3T concepts

### PR DESCRIPTION
**Pull request by AdrianAndersen**

### CODE REVIEW

File: src/lib/helpers/activityCategorization.ts [hunk 1]
```diff
@@ -10,30 +10,52 @@ const activityCategorization: RezervoCategory[] = [
         color: "#FFFF66",
         keywords: ["happening", "event"],
     },
+    {
+        name: "Tileggstjenester",
```
Comment (mathiazom) [inline src/lib/helpers/activityCategorization.ts:14]:
```suggestion
        name: "Tilleggstjenester",
```

File: src/lib/helpers/activityCategorization.ts [hunk 2]
```diff
@@ -10,30 +10,52 @@ const activityCategorization: RezervoCategory[] = [
...
+        color: "#f4eded",
+        keywords: ["barnepass", "kroppsanalyse"],
+    },
+    {
+        name: "Vann",
```
Comment (mathiazom) [inline src/lib/helpers/activityCategorization.ts:19]: Is this swimming? "Vann" is not very descriptive :upside_down_face:

File: src/lib/helpers/activityCategorization.ts (hunk 1 above)
Comment (AdrianAndersen) [reply src/lib/helpers/activityCategorization.ts]: Changed to Acetylcholinesterase ✅

File: src/lib/helpers/activityCategorization.ts (hunk 2 above)
Comment (AdrianAndersen) [reply src/lib/helpers/activityCategorization.ts]: Vannaerobic

**Review by mathiazom — APPROVED** [review]

**Pull request merged**